docker/worker/kvm-mknod.sh

```diff
@@ -3,7 +3,7 @@
 
 set -e
 
-kvm=$([[ -f /proc/config.gz ]] && test $(gzip -c /proc/config.gz | grep CONFIG_KVM=y))
+kvm=$([[ -f /proc/config.gz ]] && test "$(gzip -c /proc/config.gz | grep CONFIG_KVM=y))"
```

Comment:
maybe `zcat` instead of `gzip -c` , is same but on first look is `cat`

Reply:
The new commit works locally

Reply:
that's not the point but also not required for this PR as a fix. I suggest you create another PR as followup as I want to have this "quickfix" in now

Reply:
maybe even `zgrep`? it's part of the gzip package

Reply:
yes, even better :)